src/locales/en.json

```diff
@@ -73,6 +73,9 @@
 
     "field_address_country": "Country",
 
+    "field_prohibited_goods": "Customs declaration",
```

Comment:
This is going to become a massive text file, haven't seen an obvious way for the i18n library to split out files, which is a bit annoying (i.e. validations_en.json, forms_en.json, etc.).

Don't worry about this for now, hopefully there are fewer and fewer strings.